docs/ibc/proposals.md

```diff
@@ -0,0 +1,41 @@
+<!--
+order: 5
+-->
+
+# Governance Proposals
+
+In uncommon situations, a highly valued client may become frozen due to uncontrollable 
```

Comment:
We should clarify what a highly valued client is (locked tokens in ICS 20, e.g.)